invideoquiz/invideoquiz.py

```diff
@@ -33,8 +33,8 @@ def get_resource_string(path):
     Retrieve string contents for the file path
     """
     path = os.path.join('public', path)
-    resource_string = pkg_resources.resource_string(__name__, path)
-    return resource_string.decode('utf8')
+    resource_string = importlib.resources.files(__package__).joinpath(path)
```

Comment:
I think we should use the [XBlocks function](https://github.com/openedx/XBlock/blob/master/xblock/utils/resources.py#L20) to load the resources at all the places now.

Like we are using [here](https://github.com/openedx/DoneXBlock/blob/61ee80bbf37c9fe4be32e455d67a5c52217e2131/done/done.py#L81)

Reply:
fixed